src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -468,14 +487,16 @@ export function setupImages(layoutInfo) {
         const newPlotIdAry= makePlotIds(tableModel.highlightedRow, tableModel.totalRows,count);
         const maxPlotIdAry= makePlotIds(tableModel.highlightedRow, tableModel.totalRows,LC.MAX_IMAGE_CNT);
 
-
         const cutoutSize = getCutoutSize(layoutInfo);
         const fluxCol = getFluxColumn(layoutInfo);
+        const timeCol = getTimeColumn(layoutInfo);
+
         newPlotIdAry.forEach( (plotId) => {
             var pv = getPlotViewById(vr,plotId);
             const rowNum= Number(plotId.substring(plotIdRoot.length));
-            const webPlotReq = converterData.webplotRequestCreator(tableModel,rowNum, cutoutSize, {fluxCol});
-            if (!pv || get(pv, ['request', 'params', 'Title']) !== webPlotReq.getTitle())  {
+            const webPlotReq = converterData.webplotRequestCreator(tableModel,rowNum, cutoutSize,
+                                                                   {fluxCol, timeCol, dataSource});
```

Comment:
all parameters passed in the brace (the last argument) are optional, and not every one is needed for each mission's plot query.